_data/taps/versions/recharge.yml

```diff
@@ -0,0 +1,22 @@
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/integration-versions/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#   [INTEGRATION] VERSIONS   #
+# -------------------------- #
+
+latest-version: "1.0"
+
+released-versions:
+  - number: "1.0"
```

Comment:
Same here - `1` instead of `1.0`. Sorry for not mentioning the change! I'll do better. 😬